modules/sso-rbac/provider.tf

```diff
@@ -0,0 +1,14 @@
+provider "kubernetes" {
```

Comment:
please remove provider blocks and related data blocks from module, instead have them inside repo where you are going to use this module